dev_guide/pod_preset.adoc

```diff
@@ -0,0 +1,565 @@
+[[dev-guide-pod-presets]]
+= Injecting Information into Pods Using Pod Presets
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A _pod preset_ is an object that injects user-specified information into pods as they are created.
+
+//tag::pod-preset[]
+[IMPORTANT]
+====
+Pod presets is a Technology Preview feature only.
```

Comment:
There needs to be a note saying that PodPreset is only available if the service catalog has been installed with `oc cluster up` or the installer.